### ISSUE TITLE
Unseparated dental root

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11427]
**Preferred term label:**
Unseparated dental root

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0006486

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/621434 "unseparated roots"

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001459
- label: Dental root fusion
- parents: [Abnormal dental root morphology (HP:0006486)](https://hpo.jax.org/app/browse/term/HP:0006486)
- pmid: PMID:24862720
- def: An anomaly of the roots of a multi-rooted tooth (such as a molar) in which the roots are not physically divided by bone or periodontal ligament space.
- comment: Root fusion may be formed either by the combining of roots because of cementum deposition with time or as a result of the failure of the Hertwig epithelial root sheath to develop or fuse in the furcation area, which is frequently found in maxillary and mandibular premolars and molars. In mandibular molars, root fusion usually presents as a C-shaped root, whereas in maxillary molars fused roots may show various root shapes, such as two or more roots being fused partially or completely.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11427
- synonyms: Unseparated dental root(NONE)|Fused dental root(NONE)
